tpm2_pytss/types.py

```diff
@@ -283,6 +284,8 @@ def __bytes__(self):
         return bytes(buf)
 
     def __str__(self):
+        """foo
+        """
         b = self.__bytes__()
```

Comment:
Seems like this sneaked in

Reply:
Which it did, I see the foo. So I have to go update that and resubmit.